pyproject.toml

```diff
@@ -123,6 +126,8 @@ __version__ = \"0.0.0\"
 
 [project.entry-points."qbraid.providers"]
 local = "metriq_gym.local.provider:LocalProvider"
+qnexus = "metriq_gym.quantinuum.provider:QuantinuumProvider"
```

Comment:
again here, we are mixing up names. Let's stick with one of either `quantinuum`, or `quantinuum-nexus`